README.md

```diff
@@ -83,6 +83,11 @@ cd sriov/tests/
 pytest -v SR_IOV_Permutation
 ```
 
+Certain test cases support multiple iterations of run. The number of interations can be specified using `--iteration` option. For example, to run the test case  `SR_IOV_MultipleVFCreation_withMTU` 10 times,
```

Comment:
The first sentence can just be "Certain test cases support multiple iterations." Also in the second sentence it should be "iterations", just a small typo.

Reply:
ack